includes/tables/class-wcpdf-number-store-list-table.php

```diff
@@ -55,54 +55,71 @@ public function __construct() {
 	 * @return string Column Name
 	 */
 	public function column_default( $item, $column_name ) {
-		switch ( $column_name ) {
-			case 'number' :
-				$value = $item->id;
-				break;
-			case 'type' :
-				$value = isset( $item->document_title ) ? esc_attr( $item->document_title ) : '-';
-				break;
-			case 'calculated_number' :
-				$value = isset( $item->calculated_number ) ? $item->calculated_number : '-';
-				break;
-			case 'date' :
-				$value = $item->date;
-				break;	
-			case 'order' :
-				if ( ! empty( $item->order_id ) ) {
-					$order        = $this->get_base_order( wc_get_order( $item->order_id ) );
+		$order = $this->get_base_order( wc_get_order( $item->order_id ) );
+		$value = '-';
+		
+		if ( ! empty( $order ) ) {
+			switch ( $column_name ) {
+				case 'number':
+					$value = $item->id;
+					break;
+				case 'type':
+					$value          = '<span class="item-number number-gapped">' . __( 'gapped', 'woocommerce-pdf-invoices-packing-slips' ) . '</span>';
+					$document_types = isset( $item->document_types ) && is_array( $item->document_types ) ? $item->document_types : array();
+					
+					// document using invoice number, eg. proforma
+					if ( count( $document_types ) > 1 ) {
+						foreach ( $document_types as $key => $doc_type ) {
+							if ( 'invoice' === $doc_type ) {
+								unset( $document_types[ $key ] );
+							}
+						}
+					}
+					
+					$document_type = reset( $document_types );
+					
+					if ( ! empty( $document_type ) ) {
+						$document_slug = str_replace( '-', '_', $document_type );
+						$saved_number  = $order->get_meta( "_wcpdf_{$document_slug}_number", true );
```

Comment:
All my invoices are marked as gapped as `_wcpdf_{$document_slug}_number` returns the formatted number and `$item->id` returns the unformatted number.